src/redux/store.js

```diff
@@ -0,0 +1,11 @@
+import { configureStore } from '@reduxjs/toolkit';
+import bookReducer from './books/books';
+import catReducer from './categories/categories';
+
+// eslint-disable-next-line
```

Comment:
- [ ] I noticed that you disabled **Eslint** . Please remove the **eslint disable** statement and try to fix all the errors you are getting. Because ` Eslint` helps developers to write clean JavaScript codes and adapt best practices when writing the code.  :+1:

Reply:
hi @codecaiine ,
Yeah I had to disable eslint for that line because eslint insisted I export the variable as default which I dont know how to.
Please I would be glad if you can suggest a way to fix it without disabling eslint.